container/container_windows.go

```diff
@@ -17,8 +17,11 @@ const (
 	containerInternalSecretMountPath = `C:\ProgramData\Docker\internal\secrets`
 	containerInternalConfigsDirPath  = `C:\ProgramData\Docker\internal\configs`
 
-	// DefaultStopTimeout is the timeout (in seconds) for the shutdown call on a container
-	DefaultStopTimeout = 30
+	// defaultStopSignal is the default syscall signal used to stop a container.
+	defaultStopSignal = "SIGTERM"
```

Comment:
Yes, it's the same as the `_unix` const now, but I decided to define it in both as it's unlikely going to change, and this keeps it together with the corresponding `defaultStopTimeout` (which _does_ differs between Unix and Windows)